.github/workflows/verify-windows.yml

```diff
@@ -4,6 +4,7 @@ on: pull_request
 
 jobs:
   verify-windows:
+    timeout-minutes: 30
```

Comment:
I like that we have a timeout now. I'd argue that we can probably make it even lower but maybe in a future PR.